src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Media/MediaSystem.cs

```diff
@@ -79,6 +79,17 @@ public static bool Startup(MediaContext mc)
                 }
                 s_refCount++;
             }
+
+             /// <summary>
+            /// Set renderOption if Hardware acceleration is enabled for RDP
+            // </summary>
+
+            bool isHardwareAccelerationEnabledForRDP = false;
+            AppContext.TryGetSwitch("System.Windows.Media.IsHardwareAccelerationEnabledForRDP", out isHardwareAccelerationEnabledForRDP);
```

Comment:
Technically the .NET naming guidelines states acronyms of 3 and more letters should be cased, Rdp as Tcp

Reply:
Thanks for pointing it out I will make the necessary changes.